src/Milestones/Block/Block.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Give\Milestones\Block;
+
+class Block {
+
+	/**
+	 * Registers Milestone block
+	 *
+	 * @since 2.9.0
+	 **/
+	public function add_block() {
```

Comment:
Please use camel-casing in `src`, also elsewhere. Also, let's rename this to `registerBlock`.

Reply:
Done! Ref: https://github.com/impress-org/givewp/pull/5224/commits/10eb44ba80c2e94859368fa8d16f307e2e0eda14